src/test/java/echoServer/HTTPResponseTest.java

```diff
@@ -0,0 +1,24 @@
+package echoServer;
+
+import echoServer.HTTP.HTTPResponse;
+import echoServer.HTTP.StatusCode;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class HTTPResponseTest {
+
+    @Test
+    void testInvalidResponseReturns() throws IOException {
```

Comment:
This test description isn't very specific.

Reply:
Updated to `testInvalidRequestReturns404HTTPResponse`